src/main/scala/edu/cmu/cs/obsidian/codegen/Util.scala

```diff
@@ -135,23 +157,24 @@ object Util {
       * with an initial value but no typing information
       *
       * @param id the identifier to be declared
-      * @param e the expression to assign the identifier to as an initial value
+      * @param e  the expression to assign the identifier to as an initial value
       * @return the Yul expression for the declaration
       */
     def decl_1exp(id: Identifier, e: Expression): VariableDeclaration = decl_nexp(Seq(id), e)
 
-    def mapObsTypeToABI(ntype: String): String = {
+    def mapObsTypeToABI(ntype: String): (String, Int) = {
```

Comment:
What does the Int represent? Add a comment. Ideally this would return an object, since then the fields could have names.

Reply:
I see it's the size from reading the code, but that is not apparent from the name.

Reply:
I've added a comment and updated the name. I'm going to leave it as returning a tuple rather than a case class for now because I think this function is a little bit of a hack and temporary, but I'm not sure how it'll end up getting rewritten.